examples/airflow/README.md

```diff
@@ -1,12 +0,0 @@
-# Description
-
```

Comment:
Why is this being deleted?

Reply:
I thought this was a typo/incorrectly added file. Do you want to keep it?

Reply:
Yup we should keep these files